.github/CODEOWNERS

```diff
@@ -0,0 +1,13 @@
+# Codeowners file by GitHub
+# Reference: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @amundsen-io/amundsen-committerswill be requested for
+# review when someone opens a pull request.
+*       @amundsen-io/amundsen-committers
```

Comment:
it could be @amundsen-io/amundsen-gremlin-committers as well.

Reply:
Oh, good idea!